product_docs/docs/biganimal/release/overview/deployment_options/postgres_admin_user.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: BigAnimal's cloud account ba_admin_user
+---
+
+Learn about what operations the pg_ba_admin user can perform and what Postgres built-in roles it has.
+
+## What pg_ba_admin can do
+The pg_ba_admin user can perform the following operations:
+- Create roles
+- Create databases
+- Read and write to all user objects (tables, views, and so on)
+- Perform all monitoring functions on the Postgres instance (it has the pg_monitor role)
+- Install supported EDB extensions (see [Supported Postgres extensions and tools](biganimal/latest/overview/extensions_tools/))
+
+## What pg_ba_admin can't do
+Using any Postgres distribution, the pg_ba_admin user can't perform the following operations that a Postgres superuser can:
+
+- Execute programs on the server (pg_ba_admin doesn't have the pg_server_programs privileges)
+- Create tablespaces
+- Read and write files on the server (pg_ba_admin doesn't have the permissions pg_read_server_files and pg_write_server_files)
+- Execute the qreplace_function
+- Define LEAKPROOF functions
+- Use portions of the DBMS package that require writing the network or the filesystem
+- Set parameters related to edb_audit
```

Comment:
We don't deploy `edb_audit` on PostgreSQL.

Reply:
We already have a section for other distros, right?

Reply:
This section applies to all distros and the next section are additional items that apply only to EPAS.